custom/icds_reports/utils/data_accessor.py

```diff
@@ -48,3 +49,46 @@ def get_program_summary_data_with_retrying(step, domain, config, now, include_te
             sleep(5)
             retry += 1
     return data
+
+
+# keeping cache timeout as 2 hours as this is going to be used
+# in some script/tool which might flood us with requests
+@icds_quickcache(['state_id', 'month'], timeout=120 * 60)
+def get_disha_api_v2_data(state_id, month):
+
+    data = DishaIndicatorViewV2.objects.filter(month=month,
```

Comment:
how about using `get` here?
you should also rescue in case there is no data and give a message.

Reply:
Good thought. Doing it

Reply:
Done. Exception handled in the caller function